examples/Makefile

```diff
@@ -33,6 +33,8 @@ ifeq (${OS},Darwin)
 shared_lib = lib${1}.dylib
 else ifeq (${OS},Linux)
 shared_lib = lib${1}.so
+else ifeq (${OS},OpenBSD)
```

Comment:
OpenBSD ;)

I'll pop that into a separate commit + PR. I don't have a Linux or Mac box to test on, so it's more or less a requirement for me to get going.